### PR TITLE
Hw12 part1

### PR DESCRIPTION
**Pull request by jeneamocan**
Homework for lesson 12 (or was it 11?), part 1

### CODE REVIEW

**Comment by mtkachenk0** [comment]
what is the purpose of your PR? there are too many non-related files

File: jenea_mocan/lesson5/homework.rb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+end
+
+a = { ivan: "334455", alex: "777", petr: "password"}
+puts "Введите login"
+login = gets.chomp.to_s
+# enter login
+puts "Введите password"
+password = gets.chomp.to_s
+# enter password
+if a[login.to_sym] = password && [login.to_sym] = login
```
Comment (ston1x) [inline jenea_mocan/lesson5/homework.rb]: You are using `=` operator which assigns values.
Use `==` for comparison

File: jenea_mocan/lesson5/homework.rb [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+puts "Добро пожаловать"
```
Comment (ston1x) [inline jenea_mocan/lesson5/homework.rb]: Indent code inside of if-else statements
```
if ...
  your code
end
```

**Comment by jeneamocan** [comment]
I've cleaner the PR from all the unnecessary stuff

**Pull request closed without merging**